### PR TITLE
`<list>` fix: Prevent rendering items from nested `<list>`

### PR DESCRIPTION
**Pull request by flochtililoch**
Render list items that are direct children of the `<list>` element, or nested under a `<items>` children node.

### CODE REVIEW

File: src/components/hv-list/index.js [hunk 1]
```diff
@@ -71,14 +71,28 @@ export default class HvList extends PureComponent<HvComponentProps, State> {
     const showScrollIndicator =
       this.props.element.getAttribute('shows-scroll-indicator') !== 'false';
 
+    const itemsContainer =
+      this.props.element.firstChild &&
```
Comment (adamstep) [inline src/components/hv-list/index.js:75]: maybe I'm missing something, but `<items>` doesn't have to be the first child, right?

Comment (flochtililoch) [reply src/components/hv-list/index.js:75]: You're thinking of this kind of markup, or did you have something else in mind?
```
<list>
  <item/>
  <item/>
  <items>
    <item/>
    <item/>
  </items>
</list>
```

Comment (flochtililoch) [reply src/components/hv-list/index.js]: I've updated to account for this use case, let me know what you think

Comment (adamstep) [reply src/components/hv-list/index.js]: Yes, there can also be many <items> elements if we’ve loaded multiple pages.

Comment (flochtililoch) [reply src/components/hv-list/index.js]: > there can also be many elements if we’ve loaded multiple pages

Can you clarify this? Does the markup above not accounting for this case?

**Comment by adamstep** [comment]
Would it relatively easy to add unit tests to account for all of these cases?

**Comment by flochtililoch** [comment]
> Would it relatively easy to add unit tests to account for all of these cases?

I'll take a look. I think I'll have to refactor it to isolate the code so that I can mock elements

**Comment by flochtililoch** [comment]
@adamstep I've added tests in 4b8cb95

**Review by adamstep — APPROVED** [review]
Thanks for the tests, it adds confidence to the functionality here in case we refactor in the future.

**Pull request merged**